frontend/src/components/Post.css

```diff
@@ -2,6 +2,16 @@
   text-align: center;
 }
 
+.upvote {
```

Comment:
Some unused CSS classes here. Just need some logic in the component to apply these depending on whether you've upvoted or downvoted.